packages/manager/modules/server-sidebar/src/order.constants.js

```diff
@@ -254,6 +254,12 @@ export const ORDER_URLS = {
     },
   },
   CA: {
+    orderDomain: {
+      CA: 'https://www.ovh.com/ca',
```

Comment:
We don't have links for now, but it's to avoid a display bug if no links are provided to the order menu.